### PR TITLE
Add contributing.md

### PR DESCRIPTION
**Pull request by pkramme**
This could help

### CODE REVIEW

**Comment by shyim** [comment]
I would add the information to the ISSUE_TEMPLATE.md, because the User cant miss the information 
What is your mind @alorma

**Comment by pkramme** [comment]
GitHub creates a big yellow box over each newly made issue. Go to
https://github.com/paulkramme/roverpi and create an issue there to see it.
I think these generell guidelines should be by its own, while the template
should help any user to actually adapt these guidelines.

**Comment by pkramme** [comment]
Do you see what i mean?

**Comment by pkramme** [comment]
This is what i mean:


IMO its a more elegant way to show the guidelines. If you include these in the template it would be published with the issue anytime and this would be overhead.

**Comment by alorma** [comment]
Ok, i broke the file view from PR u_u

I will review it later

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributing
+Thank you for contributing to this project!
+
+TL;DR
```
Comment (alorma) [inline CONTRIBUTING.md:4]: Why? XD

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Contributing Issues
+- Title:
+  - Be short and specific
+  - Add a word like 'Add' or 'Fix' to the beginning and capitalize it
```
Comment (alorma) [inline CONTRIBUTING.md]: I prefer the idea of: `[Add]` or `[Fix]`

Comment (alorma) [inline CONTRIBUTING.md]: If it's fix, please add issue number in the title

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  - Do not add points, questionmarks to the title
+- Body:
+  - Do not tell your live story
+  - One picture, GIF or video tells more than a thousand words
+  - Be friendly
+  - Include nessary informations like versions, steps to reproduce, anything that could help
+- Comments:
+  - This is a discussion feature, use it
+  - BE FRIENDLY. You're the one how wants something, always keep that in mind
+  - If you email-reply watch out that you're not sending anything more than your text
```
Comment (alorma) [inline CONTRIBUTING.md:18]: :+1:

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## Issues
+### Issue Title
+The issue title represents the issue, so please choose a name that fits. You cannot assign any tags 
+by yourself, but adding a word like 'Add' or 'Fix' as the first word really help to oversee the issues.
+Also, it's important to keep the name short enough to be readable as a whole without breaking the line.
```
Comment (alorma) [inline CONTRIBUTING.md]: Title

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+Not needed, but nice to have: Captitalize the first word. It looks better.
+### Issue Body
+Here you can be more specific as you were in the title, so please explain what has happened or what
+you want to add. Please write whole, grammaticaly correct, sentences and if you can, format your texts with 
+markdown. Also remember: One picture says more than a thousand words. In issues pics, gifs or videos can really help
+and are better then 'this thing there is broken. go fix it'. It also helps if you have a bit knowledge about what you're talking when you specify the problem. For example the taskbar on you PC is not called "The thing down there where you can start your apps". 
+### Comments
```
Comment (alorma) [inline CONTRIBUTING.md:33]: Add section for reactions (:+1:, :-1:, etc), so if you read another issue and you agree with it, should add a reaction

**Comment by alorma** [comment]
I've added some comments :)

File: CONTRIBUTING.md (hunk 4 above)
Comment (pkramme) [inline CONTRIBUTING.md]: What do you mean?

Comment (alorma) [inline CONTRIBUTING.md]: Instead of name, title, you are talking about title all the time

File: CONTRIBUTING.md (hunk 2 above)
Comment (pkramme) [inline CONTRIBUTING.md]: It was more meant like "Fix this"

**Comment by pkramme** [comment]
I've updated with your critics

**Comment by alorma** [comment]
DONE!

**Pull request merged**